src/Logging.AzureAppServices/src/AzureBlobLoggerOptions.cs

```diff
@@ -21,10 +22,7 @@ public string BlobName
         get { return _blobName; }
         set
         {
-            if (string.IsNullOrEmpty(value))
-            {
-                throw new ArgumentException($"{nameof(BlobName)} must be non-empty string.", nameof(value));
-            }
+            ArgumentThrowHelper.ThrowIfNullOrEmpty(value);
```

Comment:
👍 I'm okay with using `value` instead of the property name in property setters since the stack trace should make it clear what `value` means in this context.